sdks/java/core/build.gradle

```diff
@@ -124,5 +121,5 @@ dependencies {
 }
 
 project.tasks.compileTestJava {
-  options.compilerArgs += ['-Xlint:-rawtypes'] // generated avro uses rawtypes without suppression
+  options.compilerArgs += ['-Xlint:-rawtypes']
```

Comment:
Can this still not be removed?

Reply:
Unfortunately, no. I tried but there are other places that generate such warnings and make compilation fail. 
So, it should be fixed by other PR (I add TODO for that).